.gitignore

```diff
@@ -0,0 +1,7 @@
+build/
+debian/changelog
+debian/clearwater-sip-stress.debhelper.log
+debian/clearwater-sip-stress.substvars
+debian/clearwater-sip-stress/
+debian/copyright
+debian/files
```

Comment:
`copyright` is auto-generated from `copyright.repo`. No idea what `files` is - it's empty for me - but it's there.